Fix errors reported by GolangCi

See golangCI report:

````
> make lint
cmd/libasciidoc/main.go:26:5: `helpTemplate` is unused (deadcode)
var helpTemplate = `
    ^
pkg/renderer/html5/delimited_block_test.go:285:35: `emtpy` is a misspelling of `empty` (misspell)
                        // asciidoctor will include an emtpy line in the `blockquote` element, I'm not sure why.
                                                       ^
pkg/types/document_attributes.go:71:40: `retrive` is a misspelling of `retrieve` (misspell)
        // if the argument is a pointer, then retrive the value it points to
                                              ^
pkg/types/grammar_types.go:1723:77: `implict` is a misspelling of `implicit` (misspell)
// NewImageAttributes returns a map of image attributes, some of which have implict keys (`alt`, `width` and `height`)
                                                                            ^
pkg/types/grammar_types.go:1761:40: `accross` is a misspelling of `across` (misspell)
// multiple times to the same footnote accross the document)
                                       ^
pkg/types/grammar_types.go:1802:21: `substituion` is a misspelling of `substitution` (misspell)
// Substitution the substituion group to apply when initializing a delimited block
                    ^
pkg/types/grammar_types.go:2182:82: `implict` is a misspelling of `implicit` (misspell)
// NewInlineLinkAttributes returns a map of image attributes, some of which have implict keys (`text`)
````

--- a/cmd/libasciidoc/main.go
+++ b/cmd/libasciidoc/main.go
@@ -14,17 +14,11 @@ func main() {
 	versionCmd := NewVersionCmd()
 	rootCmd.AddCommand(versionCmd)
 	rootCmd.SetHelpCommand(helpCommand)
-	// rootCmd.SetHelpTemplate(helpTemplate)
-	// rootCmd.PersistentFlags().BoolP("help", "h", false, "Print usage")
-	// rootCmd.PersistentFlags().MarkShorthandDeprecated("help", "please use --help")
 	if err := rootCmd.Execute(); err != nil {
 		fmt.Println(err)
 		os.Exit(1)
 	}
 }
-
-// var helpTemplate = `
-// {{if or .Runnable .HasSubCommands}}{{.UsageString}}{{end}}`
 
 var helpCommand = &cobra.Command{
 	Use:               "help [command]",

--- a/cmd/libasciidoc/main.go
+++ b/cmd/libasciidoc/main.go
@@ -23,8 +23,8 @@ func main() {
 	}
 }
 
-var helpTemplate = `
-{{if or .Runnable .HasSubCommands}}{{.UsageString}}{{end}}`
+// var helpTemplate = `
+// {{if or .Runnable .HasSubCommands}}{{.UsageString}}{{end}}`
 
 var helpCommand = &cobra.Command{
 	Use:               "help [command]",

--- a/pkg/parser/delimited_block_test.go
+++ b/pkg/parser/delimited_block_test.go
@@ -580,7 +580,7 @@ foo
 ====`
 			expectedResult := types.DelimitedBlock{
 				Attributes: types.ElementAttributes{
-					types.AttrKind: types.Example,
+					types.AttrKind:  types.Example,
 					types.AttrTitle: "example block title",
 				},
 				Elements: []interface{}{

--- a/pkg/renderer/html5/delimited_block_test.go
+++ b/pkg/renderer/html5/delimited_block_test.go
@@ -464,7 +464,7 @@ are preserved, but not trailing spaces</p>
 			actualContent := `[quote]
 ____
 ____`
-			// asciidoctor will include an emtpy line in the `blockquote` element, I'm not sure why.
+			// asciidoctor will include an empty line in the `blockquote` element, I'm not sure why.
 			expectedResult := `<div class="quoteblock">
 <blockquote>
 

--- a/pkg/types/document_attributes.go
+++ b/pkg/types/document_attributes.go
@@ -68,7 +68,7 @@ func (a DocumentAttributes) Add(key string, value interface{}) {
 	}
 	v := reflect.ValueOf(value)
 	k := v.Kind()
-	// if the argument is a pointer, then retrive the value it points to
+	// if the argument is a pointer, then retrieve the value it points to
 	if k == reflect.Ptr {
 		if v.Elem().IsValid() {
 			a[key] = v.Elem().Interface()

--- a/pkg/types/grammar_types.go
+++ b/pkg/types/grammar_types.go
@@ -1720,7 +1720,7 @@ func NewImageMacro(path string, attributes ElementAttributes) (ImageMacro, error
 	}, nil
 }
 
-// NewImageAttributes returns a map of image attributes, some of which have implict keys (`alt`, `width` and `height`)
+// NewImageAttributes returns a map of image attributes, some of which have implicit keys (`alt`, `width` and `height`)
 func NewImageAttributes(alt, width, height interface{}, otherAttrs []interface{}) (ElementAttributes, error) {
 	result := ElementAttributes{}
 	var altStr, widthStr, heightStr string
@@ -1758,7 +1758,7 @@ func ResetFootnoteSequence() {
 }
 
 // Footnote a foot note, without or without explicit reference (an explicit reference is used to refer
-// multiple times to the same footnote accross the document)
+// multiple times to the same footnote across the document)
 type Footnote struct {
 	ID int
 	// Ref the optional reference
@@ -1799,7 +1799,7 @@ type DelimitedBlock struct {
 	Elements   []interface{}
 }
 
-// Substitution the substituion group to apply when initializing a delimited block
+// Substitution the substitution group to apply when initializing a delimited block
 type Substitution func([]interface{}) ([]interface{}, error)
 
 // None returns the content as-is, but nil-safe
@@ -2179,7 +2179,7 @@ func (l InlineLink) Text() string {
 // AttrInlineLinkText the link `text` attribute
 const AttrInlineLinkText string = "text"
 
-// NewInlineLinkAttributes returns a map of image attributes, some of which have implict keys (`text`)
+// NewInlineLinkAttributes returns a map of image attributes, some of which have implicit keys (`text`)
 func NewInlineLinkAttributes(text interface{}, otherAttrs []interface{}) (ElementAttributes, error) {
 	result := ElementAttributes{}
 	var textStr string

